### DISCUSSION TITLE
Room code lifetime

### DISCUSSION THREAD

**Discussion by bemyevent** [mozilla/hubs#3814] [General Questions & Discussion] [answered]
Hello 
Do you know how to increase room s code to enter in a room lifetime ? For the moment I think it is 72h , cow van increase the value ? 
Thx

**Comment by robinkwilson** [comment, marked as answer]
The room ID (the one in the full URL) is permanent. The one in the "invite" dialog expires. 

Unfortunately the invite dialog code is hard-coded on the backend, so the upper limit is 72h for now. 

If you make an issue/feature request that would be helpful if you need that kind of functionality! https://github.com/mozilla/hubs/issues

**Reply by robinkwilson** [reply]
Marking as answered for now, if this doesn't answer your question please unmark!
